zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/authorization/DbMappingState.java

```diff
@@ -78,6 +78,18 @@ public void addRole(final long mappingKey, final long roleKey) {
     }
   }
 
+  @Override
+  public void addTenant(final long mappingKey, final String tenantId) {
+    this.mappingKey.wrapLong(mappingKey);
+    final var fkClaim = claimByKeyColumnFamily.get(this.mappingKey);
+    if (fkClaim != null) {
+      final var fkKey = fkClaim.inner();
```

Comment:
🔧 I would rename `fkKey` to `claim`. `fk` is used to refer to something that's a foreign key. Here you have a composite key of 2 strings.